### PR TITLE
FEATURE:  Add ContactForm NodeType based on the Neos.Fusion.Form package

### PR DESCRIPTION
**Pull request by mficzel**
This change adds a very basic ContactForm NodeType based on Neos.Fusion.Forms.  https://github.com/neos/fusion-form

The content still mentions the fusion.form framework alongside neos.fusion.form but the content o the form page is now a fusion form with inspector settings.

### CODE REVIEW

**Comment by mficzel** [comment]
This needs the following bugfix in fusion forms https://github.com/neos/fusion-form/pull/47

File: Resources/Private/Fusion/Content/ContactForm.fusion [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            message {
+                type = 'Neos.Fusion.Form.Runtime:Message'
+                options.message = ${q(node).property('message')}
+            }
+
+            email {
+                @if.hasRecipientMail = ${q(node).property('emailAddress')}
+                type = 'Neos.Fusion.Form.Runtime:Email'
+                options {
+                    senderAddress = ${data.email}
```
Comment (jonnitto) [inline Resources/Private/Fusion/Content/ContactForm.fusion:51]: I wouldn't set the entered E-Mail as sender. Better is to add a reply to address

**Review by jonnitto — CHANGES_REQUESTED** [review]

File: Resources/Private/Fusion/Content/ContactForm.fusion (hunk 1 above)
Comment (mficzel) [reply Resources/Private/Fusion/Content/ContactForm.fusion]: adjusted as requested.

File: Resources/Private/Fusion/Content/ContactForm.fusion [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+            schema {
+                name = ${Form.Schema.string().isRequired()}
+                email = ${Form.Schema.string().isRequired().validator('EmailAddress')}
+                message = ${Form.Schema.string().isRequired()}
+            }
+        }
+
+        action {
+            message {
```
Comment (Sebobo) [inline Resources/Private/Fusion/Content/ContactForm.fusion:42]: This one would have no effect if the redirect is enabled right?

Comment (mficzel) [reply Resources/Private/Fusion/Content/ContactForm.fusion:42]: Yes, the message will be in the result but the location header will cause an immediate jump.

**Comment by Sebobo** [comment]
I would say we should then throw out the old form. IMO this is just confusing having the yaml still there.

**Review by Sebobo — APPROVED** [review]
Looks good by reading 👍

**Comment by mficzel** [comment]
@Sebobo i removed the form nodetype templates and settings, this also allowed to remove the dependency to nodetypes-form.

**Comment by mficzel** [comment]
@jonnitto @markusguenther how about a second review from one of you

**Review by jonnitto — APPROVED** [review]

**Pull request merged**